src/gsCore/gsDofMapper.h

```diff
@@ -314,6 +324,16 @@ class GISMO_EXPORT gsDofMapper
         return MAPPER_PATCH_DOF(i,k) + m_numCpldDofs - m_numFreeDofs;
     }
 
+    /// @brief Returns the tagged dof index
+    inline index_t tindex( index_t i, index_t k = 0 ) const
+    {
+        GISMO_ASSERT(m_curElimId==0, "finalize() was not called on gsDofMapper");
+        if (m_tagged.empty())
+            return MAPPER_PATCH_DOF(i,k) + m_numCpldDofs - m_numFreeDofs;
+        else
+            return std::distance(m_tagged.begin(),std::lower_bound(m_tagged.begin(),m_tagged.end(),MAPPER_PATCH_DOF(i,k)));
```

Comment:
Shoud be just
return std::distance(m_tagged.begin(),std::lower_bound(m_tagged.begin(),m_tagged.end(),MAPPER_PATCH_DOF(i,k))); 

without if/else i think..

Reply:
Ups, you are right